src/test/java/org/opensearch/knn/plugin/script/KNNScoringScriptEngineTests.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.plugin.script;
+
+import java.io.IOException;
+
+import org.opensearch.knn.KNNTestCase;
+import org.opensearch.script.ScriptEngine;
+
+public class KNNScoringScriptEngineTests extends KNNTestCase {
+
+    public void testGetSupportedContexts() throws IOException {
+        try (ScriptEngine engine = new KNNScoringScriptEngine()) {
+            assertNotNull(engine.getSupportedContexts());
```

Comment:
Shall we also check it is not empty or ScoreScript.CONTEXT is exists?

Reply:
Non-empty makes sense. I'll make that change.  As for specifying the context, I think that gets too far into implementation details that may change.  Anything other than ScoreScript.CONTEXT would throw an exception on `compile()` anyway, which I hope is tested elsewhere.